src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -182,28 +202,31 @@ export class SkyTabsetComponent
     });
   }
 
-  public ngAfterViewInit() {
+  public ngAfterViewInit(): void {
     this.adapterService.init(this.elRef);
 
-    this.adapterService.overflowChange.subscribe((currentOverflow: boolean) => {
-      this.updateDisplayMode(currentOverflow);
-    });
+    this.adapterService.overflowChange
```

Comment:
This looks like this fixes #41. However, if you pull this build into `skyux2-docs`, click the `Development` tab and shrink the window down to a small horizontal width, the tabs never collapse like they should.

Reply:
I was able to see the tabs collapse on the docs repo. However, the collapse didn't happen until the window was smaller than it should have been. I noticed that there's some negative margin on the docs' tabset, which we can fix in another pull request. Am I missing something obvious?

Reply:
I adjusted the logic a bit to accommodate the left-hand margin of the first tab when running the calculations.